CONTRIBUTING.rst

```diff
@@ -49,3 +50,28 @@ You want to make some kind of change to the code base
 10. create the pull request, e.g. following the instructions `here <https://help.github.com/articles/creating-a-pull-request/>`_.
 
 In case you feel like you've made a valuable contribution, but you don't know how to write or run tests for it, or how to generate the documentation: don't let this discourage you from making the pull request; we can help you! Just go ahead and submit the pull request, but keep in mind that you might be asked to append additional commits to your pull request.
+
+You want to make a release
+**************************
+
+Section is for maintainers of the package.
+
+1. Determine what new version to use. Package uses `semantic versioning <https://semver.org>`_.
+2. Checkout ``HEAD`` of ``main`` branch with ``git fetch`` and ``git checkout main``.
+3. Update version in
+  1. ewatercycle/version.py
+  2. docs/conf.py
+4. Update CHANGELOG.rst with changes between current and new version.
```

Comment:
I am wondering if citaion.cff should include version as well.

Reply:
Preferred field is a doi (Zenodo concept DOI), instead of version, date-released or commit. We will add it once first GH release has been made.